### DISCUSSION TITLE
v0.7.0

### DISCUSSION THREAD

**Discussion by Hayao-H** [Hayao-H/Niconicome#352] [一般]
# 概要
- 予定より早いですがリリースです 💨 
- 今回追加された動画の高さ・カラムの非表示等は「設定＞スタイル設定＞スタイルファイルの書き出し」で出力される実行ディレクトリ以下の「chrome\userChrome.json」を編集することで設定出来ます。
**※ホットリロード🔥 に対応しておりますが、VSDodeだと失敗します。**

## 追加
- ファイルのサフィックス機能を追加
- 取得するサムネイルのサイズを指定出来るようにした #336
- シリーズ取得機能を追加 #335
- ファイルフォーマットに```<owner-id>```を追加　#306
-保存フォルダーの自動マッピング機能を追加 #332
> デフォルトフォルダーに```<autoMap>```を付加すると動作
- FFmpegのフォーマット指定機能を追加 #304
- カラムの非表示機能を追加 #328
- 動画リストの高さ設定機能を追加 #325
- ブックマーク機能を追加 #327
- 動画情報のコピー機能を追加 #317
- DL時の終了前確認機能を追加 #321
- アプリの再起動・終了機能を追加 #320
- ログファイルへのアーキテクチャー出力を追加 #305
- ウィンドウ数を制限する機能を追加 #315
- 履歴機能・揮発性プレイリスト機能を追加 #280
- 選択動画数などを表示する機能を追加 #303
## 修正
- 動画リストの一番上までスクロール出来ない問題を修正 #311
- XMLファイルのエンコード表示の問題を修正（宣言そのものを削除） #308
- 投コメをDL出来ない問題を修正 #310
- ファイル名が変わると動画ファイルを追跡出来なくなる問題を修正（別途オプションを有効にする必要有り #300
- 動画を削除出来ない問題を修正 #323
- ダークテーマを改善 #318
- すべての過去ログをDL出来ない問題を修正 #313
- あとからキューに追加した動画でも自動でDLするようにした #278
- 並び替え後にリストがからになる問題を修正 #284

<hr /><em>This discussion was created from the release <a href='https://github.com/Hayao-H/Niconicome/releases/tag/v0.7.0'>v0.7.0</a>.</em>

**Comment by T-10001** [comment]
新しいレリース（ｖ0.7.0）を開く時エラーが発生する
![image](https://user-images.githubusercontent.com/20717627/123888643-46708380-d997-11eb-8098-124db279b770.png)

**Comment by Hayao-H** [comment]
ご報告ありがとうございます。
修正しました。#353
https://github.com/Hayao-H/Niconicome/releases/tag/v0.7.1
